server/internal/config_parser.go

```diff
@@ -25,100 +25,50 @@ type Configuration struct {
 
 // Server struct to hold server's information
 type Server struct {
-	Host string `json:"host"`
-	Port string `json:"port"`
+	Host string `json:"host" validate:"nonzero"`
+	Port string `json:"port" validate:"nonzero"`
 
-	RedisHost string `json:"redisHost"`
-	RedisPort string `json:"redisPort"`
+	RedisHost string `json:"redisHost" validate:"nonzero"`
+	RedisPort string `json:"redisPort" validate:"nonzero"`
 	RedisPass string `json:"redisPass"`
 }
 
 // MailSender struct to hold sender's email, password
 type MailSender struct {
-	Email       string `json:"email"`
-	SendGridKey string `json:"sendgrid_key"`
-	Timeout     int    `json:"timeout"`
+	Email       string `json:"email" validate:"nonzero"`
+	SendGridKey string `json:"sendgrid_key" validate:"nonzero"`
+	Timeout     int    `json:"timeout" validate:"min=30"`
 }
 
 // DB struct to hold database file
 type DB struct {
-	File string `json:"file"`
+	File string `json:"file" validate:"nonzero"`
 }
 
 // JwtToken struct to hold JWT information
 type JwtToken struct {
-	Secret  string `json:"secret"`
-	Timeout int    `json:"timeout"`
+	Secret  string `json:"secret" validate:"nonzero"`
+	Timeout int    `json:"timeout" validate:"min=5"`
 }
 
 // GridAccount struct to hold grid account mnemonics
 type GridAccount struct {
-	Mnemonics string `json:"mnemonics"`
-	Network   string `json:"network"`
+	Mnemonics string `json:"mnemonics" validate:"nonzero"`
+	Network   string `json:"network" validate:"nonzero"`
 }
 
 // ReadConfFile read configurations of json file
-func ReadConfFile(path string) ([]byte, error) {
-	confFile, err := os.Open(path)
-	if err != nil {
-		return nil, err
-	}
-
-	defer confFile.Close()
-	conf, err := io.ReadAll(confFile)
+func ReadConfFile(path string) (Configuration, error) {
+	config := Configuration{NotifyAdminsIntervalHours: 6, BalanceThreshold: 2000}
```

Comment:
Not sure why these values exist here, or that's for default values?

Reply:
they are the default ones